### PR TITLE
feat: add jobOwnerId as metadata on export results

### PR DESCRIPTION
**Pull request by carvantes**
set `jobOwnerId` as metadata on export results s3 files

Related PRs:
- https://github.com/awslabs/fhir-works-on-aws-persistence-ddb/pull/124

Checklist:

* [x] Have you successfully deployed to an AWS account with your changes?
* [n/a] Have you written new tests for your core changes, as applicable?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: bulkExport/glueScripts/export-script.py [hunk 1]
```diff
@@ -23,7 +23,7 @@
 glueContext = GlueContext(SparkContext.getOrCreate())
 job = Job(glueContext)
 
-args = getResolvedOptions(sys.argv, ['JOB_NAME', 'jobId', 'exportType', 'transactionTime', 'since', 'outputFormat', 'ddbTableName', 'workerType', 'numberWorkers', 's3OutputBucket'])
+args = getResolvedOptions(sys.argv, ['JOB_NAME', 'jobId', 'jobOwnerId', 'exportType', 'transactionTime', 'since', 'outputFormat', 'ddbTableName', 'workerType', 'numberWorkers', 's3OutputBucket'])
```
Comment (rsmayda) [inline bulkExport/glueScripts/export-script.py:26]: What are our concerns with existing exports that don't have jobOwner? Do exports have a TTL?

Comment (carvantes) [reply bulkExport/glueScripts/export-script.py:26]: Export results are available for 3 days after the job finishes: 
https://github.com/awslabs/fhir-works-on-aws-deployment/blob/d975e7b33cb3f8f0c2b91951f141303874d46d75/cloudformation/bulkExport.yaml#L83-L90

However, the `jobOwnerId` metadata is currently unused. Nothing in FWoA will break if an slightly old export results file is missing the `jobOwnerId` metadata. (FWoA just cares about the jobId and retrieves the other export metadata from DDB)

Customers who build a feature relying on the `jobOwnerId` metadata do need to be mindful that `jobOwnerId` may be missing on older files. But in practice this edge case can most likely just be disregarded as 3 days is a small time window.

Comment (carvantes) [reply bulkExport/glueScripts/export-script.py:26]: Export results are available for 3 days after the job finishes: 
https://github.com/awslabs/fhir-works-on-aws-deployment/blob/d975e7b33cb3f8f0c2b91951f141303874d46d75/cloudformation/bulkExport.yaml#L83-L90

However, the `jobOwnerId` metadata is currently unused. Nothing in FWoA will break if an slightly old export results file is missing the `jobOwnerId` metadata. (FWoA just cares about the jobId and retrieves the other export metadata from DDB)

Customers who build a feature relying on the `jobOwnerId` metadata do need to be mindful that `jobOwnerId` may be missing on older files. But in practice this edge case can most likely just be disregarded as 3 days is a small time window.

Comment (rsmayda) [reply bulkExport/glueScripts/export-script.py:26]: Thanks for confirming

**Review by rsmayda — APPROVED** [review]

**Pull request merged**